src/AliasingContainer.php

```diff
@@ -3,7 +3,8 @@
 namespace Dhii\Container;
 
 use Dhii\Container\Util\StringTranslatingTrait;
-use Psr\Container\ContainerInterface;
+use Dhii\Data\Container\ContainerInterface;
```

Comment:
Did I point out that this change needs to be made? 🤔 I legit don't remember. [`dhii/data-container-interface`][] is deprecated.

[`dhii/data-container-interface`]: https://github.com/Dhii/data-container-interface

Reply:
Here: https://github.com/Dhii/containers/pull/2#discussion_r285380423

Reply:
The reason it's deprecated is that many of the things from that package are now part of `dhii/collection-interface`. Some, like the additional getters for exceptions, are not IIRC. They look useful, but their utility has never actually been put to use, AFAIK.

Reply:
Ya, OK, thanks. Sorry about that! Could you roll back please? If you want, you can implement some of the interfaces from collections, if you think it makes stuff more useful.

Reply:
I'm of the opinion that the PSR container interface is enough. That's what they implemented before, and they worked. No point fixing something that's not broken.

Reply:
Do I remove the `dhii/data-container-interface` dependency while I'm at it?

Reply:
Yes, please, kind sir!